ast/ast.go

```diff
@@ -56,15 +58,38 @@ func (g *Grammar) String() string {
 	return buf.String()
 }
 
+// NullableVisit recursively determines whether an object is nullable
```

Comment:
Please make the documentation comments full sentences (including a full stop at the end).

```suggestion
// NullableVisit recursively determines whether an object is nullable.
```

Reply:
Fixed